typescript/system.ts

```diff
@@ -0,0 +1,13 @@
+export interface System {
```

Comment:
Whats the identifier used to reference an instance of one of these?

Reply:
Hm, not sure how we store it yet. Guess we need a table, but I only envisioned one record. Leave it with me over lunch.